api/modules/dataSyncs.js

```diff
@@ -12,6 +13,45 @@ const { GITHUB, TOGGL } = require('../config/credentials')
 
 const dataSyncs = module.exports = (() => {
 
+    const importInvoicelyCsvToStripe = async () => {
+        const invoiceFile = INVOICELY_CSV_PATH
+        const csvFile =
+            await amazon.fetchFile({ file: invoiceFile })
+                .then(file => {
+                    return invoicelyCodebase.modelCSV(file)
+                })
+                .catch(err => {
+                    console.log('error', err);
+                    return err.message
+                })
+        const stripeCustomers = await stripe.listAllCustomers()
+        const customersFromCsv = []
+        try {
+            csvFile.map(async csvData => {
+                let customerInformation
+                const customer = stripeCustomers.data.find(customerData => customerData.name == csvData.Client)
+                csvData.Total = csvData.Total.replace(/,/g, '')
+                const amount = parseInt(csvData.Total) * 100
```

Comment:
Probably would be good to add a `const amount = Number((parseInt(csvData.Total) * 100).toFixed(0))` to avoid floating point errors

Also might as well use `Number` over `parseInt`

```
const amount = Number((Number(csvData.Total) * 100).toFixed(0))
```